contents/docs/surveys/creating-surveys.mdx

```diff
@@ -50,6 +50,10 @@ There are three options for displaying a survey:
 
 2. **API:** Implement your own survey UI and use PostHog to handle display logic, capturing results, and analytics.
 
+3. **Feedback button:** Set up a survey based on your own custom button or our prebuilt feedback tab.
```

Comment:
Text says 'There are three options' but only list 2

Reply:
Created https://github.com/PostHog/posthog/issues/27764 to figure out why its in beta